config/config.yaml

```
@@ -80,10 +105,23 @@
     enabled: true
     stale_threshold: 24h  # How long to keep models in memory after last seen
     cleanup_interval: 10m  # How often to check for stale models
+  
+  # Model routing strategy (v0.0.16+)
+  # Controls how requests are routed when models aren't available on all endpoints
+  routing_strategy:
+    type: "strict"  # Options: strict, optimistic, discovery
+    options:
+      # Fallback behavior when model not found (optimistic mode)
+      fallback_behavior: "compatible_only"  # Options: compatible_only, all, none
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)